### DISCUSSION TITLE
Device hangs when powered up with power connected

### DISCUSSION THREAD

**Discussion by ignusfast** [eduardofilo/RG350_adam_image#117] [Hardware problems?]
I'm assuming this issue might be related to my particular RG350...

Only with the Adam image, if the device is connected to USB-C power and I try to turn it on, it completely hangs with the power LED being super dimly lit.  Holding power does nothing; holding Reset makes the LED get slightly brighter, but it goes dim again when I let go.  No combination of buttons held for any length of time will reset it.  I actually have to remove the back and disconnect/reconnect the battery.

If the RG350 isn't connected to power, it comes up just fine.  Once it's running, I can disconnect/reconnect it from power with no issue, and it charges just fine.

Again, it did NOT do this with the native image, but I cannot be certain it's not just flaking out...

**Comment by eduardofilo** [comment]
Hello. It does seem like a problem related to your unit, because I precisely have an original RG350 and I do not see the problem. Besides that it is the first case that I hear and I know that the image is used in that machine. Try to do the test with an image of the stock system, with the 2021-10-15 version of ODbeta (this is the version used in Adam) and the last one (2021-11-25) to isolate the problem and to discard configuration issues in Adam. Also check the voltage that the battery has under different conditions. At the end of this paragraph (in spanish, but the table can be understand easily) you have different values of a functional battery:

https://apuntes.eduardofilo.es/retro-emulacion/rg-350.html#bateria

**Comment by eduardofilo** [comment]
Also try with different microSD cards, just in case the voltage at which the console is operating when it works with batteries is not to the liking of your card, since the fact that the LED light dim indicates that the console has not found a valid boot on the microSD. It shouldn't influence as the console has voltage regulators after the battery, but test anyway.
